src/components/calcite-value-list/resources.ts

```diff
@@ -0,0 +1,7 @@
+export const CSS = {
+  container: "container",
+  dragHandle: "handle"
```

Comment:
sand: Maybe rename `handle` instead of `dragHandle`

Reply:
Done. 👍